worlds/apquest/client/ap_quest_client.py

```diff
@@ -0,0 +1,262 @@
+import asyncio
+import sys
+from argparse import Namespace
+from enum import Enum
+from typing import TYPE_CHECKING, Any
+
+from CommonClient import CommonContext, gui_enabled, logger, server_loop
+from NetUtils import ClientStatus
+
+from ..apquest.events import ConfettiFired, LocationClearedEvent, VictoryEvent
+from ..apquest.game import Game, Input
+from ..apquest.items import Item
+from ..apquest.locations import Location
+from .game_manager import APQuestManager
+from .graphics import PlayerSprite
+from .item_quality import get_quality_for_network_item
+from .sounds import CONFETTI_CANNON, ITEM_JINGLES, VICTORY_JINGLE
+
+if TYPE_CHECKING:
+    import kvui
+
+
+# !!! IMPORTANT !!!
+# The client implementation is *not* meant for teaching.
+# Obviously, it is written to the best of its author's abilities,
+# but it is not to the same standard as the rest of the apworld.
+# Copy things from here at your own risk.
+
+
+class ConnectionStatus(Enum):
+    NOT_CONNECTED = 0
+    SCOUTS_NOT_SENT = 1
+    SCOUTS_SENT = 2
+    GAME_RUNNING = 3
+
+
+class APQuestContext(CommonContext):
+    game = "APQuest"
+    items_handling = 0b111  # full remote
+
+    client_loop: asyncio.Task[None]
+
+    last_connected_slot: int | None = None
+
+    slot_data: dict[str, Any]
+
+    ap_quest_game: Game | None = None
+    hard_mode: bool = False
+    hammer: bool = False
+    extra_starting_chest: bool = (False,)
```

Comment:
```suggestion
    extra_starting_chest: bool = False
```
Not a bool, and based on other uses I think it's actually the value that's wrong on this one?